spock/plugins/helpers/inventory.py

```diff
@@ -144,6 +145,16 @@ def __init__(self, ploader, settings):
         # stores the last click action for confirmation
         self.last_click = None
 
+        # to know when the inventory got synced with server
+        self.is_synchronized = False
```

Comment:
Ah sorry, I actually misread the diff and thought it wasn't being set in ``__init__``!